quickstarts/microsoft.machinelearningservices/machine-learning-compute-create-computeinstance/azuredeploy.parameters.json

```diff
@@ -10,6 +10,9 @@
         },
         "objectId": {
             "value": "GEN-AZUREAD-OBJECTID"
-        }
+        },
+        "inlineCommand": {
```

Comment:
remove this since you have a defaultValue

Reply:
Done